assets/css/menu.scss

```
@@ -31,4 +31,30 @@
 		position: relative;
 		top: 1px;
 	}
+
+
+	/**
+	 * Adjust upsells styles
+	 */
+	.wpjm-addon-upsell__badge {
+		display: flex;
+		vertical-align: middle;
+		box-sizing: border-box;
+		margin: auto 0;
+		padding: 0 3px;
+		min-width: 15px;
+		height: 15px;
```

One last fix from the conversation p1700522161547909/1700490939.485889-slack-C9W0QF6KA

```suggestion
		min-width: 16px;
		height: 16px;
```